log/multi_logger.go

```diff
@@ -0,0 +1,88 @@
+package log
+
+import (
+	"io"
+
+	"github.com/apex/log"
+)
+
+var _ Logger = &MultiLogger{}
+
+type MultiLogger struct {
+	defaultLogger *DefaultLogger
+	infoLogger    *DefaultLogger
```

Comment:
Maybe there are better names than "default" and "info". I thought of "verbose" instead of "info", but wasn't sure.